insights/specs/jdr_archive.py

```diff
@@ -0,0 +1,45 @@
+from functools import partial
+from insights.specs import Specs
+from insights.core.plugins import datasource
+from insights.core.context import JDRContext
+from insights.core.spec_factory import simple_file, foreach_collect, first_file, glob_file, listdir
+
+first_file = partial(first_file, context=JDRContext)
+glob_file = partial(glob_file, context=JDRContext)
+simple_file = partial(simple_file, context=JDRContext)
+foreach_collect = partial(foreach_collect, context=JDRContext)
+
+
+class JDRSpecs(Specs):
+    # Here use glob_file is to return list type result to keep same with  jboss specs default.py
```

Comment:
@chenlizhong please add docstrings to this module and class so they will be included in the documentation.  You will need to also add a reference to this module in `docs/api_index.rst`.